2022-01-28 15:58:17 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(1) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:112) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:21) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:53) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:19) [classes/:?]
2022-01-28 16:47:00 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(1) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:112) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:21) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:128) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:53) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:19) [classes/:?]
2022-01-28 19:23:54 ERROR Something went wrong 
org.postgresql.util.PSQLException: Can't use query methods that take a query string on a PreparedStatement.
	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:101) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.findAll(TicketDAO.java:65) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.getTickets(DBRepository.java:28) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.getTickets(TicketBL.java:28) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.viewTicket(MainMenu.java:154) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:65) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 16:54:50 ERROR Can't get connection 
2022-01-29 17:18:55 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(56) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:19:48 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(34) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:27:53 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(9) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:33:05 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(56) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:38:19 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(34) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:40:05 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(34) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:41:50 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(45) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:46:48 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(0) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 17:48:21 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(0) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 18:19:40 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(56) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 18:20:46 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(78) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-29 18:22:42 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(56) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:89) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:131) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:44) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-31 12:44:05 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(7) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:106) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:136) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:48) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-31 12:53:14 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(10) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:106) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:136) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:48) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
2022-01-31 14:07:41 ERROR Something went wrong 
org.postgresql.util.PSQLException: ERROR: insert or update on table "tickets" violates foreign key constraint "tickets_employee_id_fkey"
  Detail: Key (employee_id)=(0) is not present in table "employees".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:162) ~[postgresql-42.3.1.jar:42.3.1]
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:151) ~[postgresql-42.3.1.jar:42.3.1]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:106) [classes/:?]
	at com.revature.reimbursement.dl.TicketDAO.add(TicketDAO.java:1) [classes/:?]
	at com.revature.reimbursement.dl.DBRepository.addTicket(DBRepository.java:22) [classes/:?]
	at com.revature.reimbursement.bl.TicketBL.addTicket(TicketBL.java:21) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.createTicket(MainMenu.java:136) [classes/:?]
	at com.revature.reimbursements.ui.MainMenu.start(MainMenu.java:53) [classes/:?]
	at com.revature.reimbursements.Driver.main(Driver.java:20) [classes/:?]
